tests/Mutator/Number/DecrementIntegerTest.php

```diff
@@ -162,24 +162,6 @@ public function provideMutationCases(): array
 if (0 == count($a)) {
     echo 'bar';
 }
-PHP
-                ,
-            ],
-            'It does decrement zero when it is compared as greater than count() on the right side' => [
```

Comment:
seems like the code still decrements `0` to `-1`, but now we don't have a test that proves it.

do we need to update the code to not decrement in this case?